protos/processor.proto

```diff
@@ -22,9 +22,11 @@ import "transaction.proto";
 
 
 // The registration request from the transaction processor to the
-// validator/executor
+// validator/executor.
```

Comment:
Typo in commit message. protocl_version -> protocol_version

Reply:
Corrected. Good catch, thanks.